modules/mobile/docs/openapi.yaml

```diff
@@ -790,6 +790,41 @@ paths:
       security:
         - Bearer: [ ]
       summary: /v0/appointment
+  /v0/awards:
+    get:
+      description: Get current awards overview
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Awards.yml
+          description: OK
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '408':
+          $ref: '#/components/responses/408'
+        '422':
+          $ref: '#/components/responses/422'
+        '500':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/AppointmentCheckinErrors500.yml
```

Comment:
If awards uses the checkin errors, maybe we should rename checkin errors. Does it also include the goofy nested errors? How would we even know that? The specs don't have any error cases.

Reply:
Ah whoops, swapped it out with the generic error